src/main/java/seedu/address/model/client/insurance/InsurancePlansManager.java

```diff
@@ -301,4 +301,18 @@ public String toString() {
 
         return plans.toString();
     }
+
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
Should we standardise this to `other` like we did in the `InsurancePlan` equals method?